### DISCUSSION TITLE
million of issues in the iptv channels list

### DISCUSSION THREAD

**Discussion by bedevlab** [iptv-org/community#604] [Q&A] [answered]
most of channels have bad quality logos, some have none, most of countries and categories have channels listed that should not be there, is there any way we can contribute to the project to fix all of that, me myself i can contribute to fix all possible mistakes and issues

**Comment by freearhey** [comment, marked as answer]
You can fix all the errors simply by editing our [iptv-org/database](https://github.com/iptv-org/database/tree/master).
